.github/workflows/build-docker.yml

```diff
@@ -263,20 +268,71 @@ jobs:
           context: .
           project: 7hd4vdzmw5 # astral-sh/uv
           platforms: linux/amd64,linux/arm64
-          push: true
+          push: ${{ needs.docker-plan.outputs.push }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          # TODO(zanieb): Annotations are not supported by Depot yet and are ignored
           annotations: ${{ steps.meta.outputs.annotations }}
```

Comment:
Hoping this is a brief problem https://github.com/depot/build-push-action/pull/38